ChangeLog

```diff
@@ -9,6 +9,9 @@ Release date: Undefined
 ..
   Put new features here and also in 'doc/whatsnew/2.8.rst'
 
+* Add new ``ConfusingConsecutiveElifChecker``. This checker emits a refactoring message (R5601 ``confusing-consecutive-elif``)
+  if if/elif statements with different indentation levels follow directly one after the other.
```

Comment:
The ChangeLog entry should probably mention that this is an optional extension.

Reply:
Changed to "This optional checker emits [...]"